cibuildwheel/windows.py

```diff
@@ -8,89 +8,72 @@
-        PythonConfiguration(version='3.4.x', arch="32", identifier='cp34-win32', path='C:\\Python34', nuget_version=None),
-        PythonConfiguration(version='3.4.x', arch="64", identifier='cp34-win_amd64', path='C:\\Python34-x64', nuget_version=None),
-        PythonConfiguration(version='3.5.x', arch="32", identifier='cp35-win32', path='C:\\Python35', nuget_version="3.5.4"),
-        PythonConfiguration(version='3.5.x', arch="64", identifier='cp35-win_amd64', path='C:\\Python35-x64', nuget_version="3.5.4"),
-        PythonConfiguration(version='3.6.x', arch="32", identifier='cp36-win32', path='C:\\Python36', nuget_version="3.6.8"),
-        PythonConfiguration(version='3.6.x', arch="64", identifier='cp36-win_amd64', path='C:\\Python36-x64', nuget_version="3.6.8"),
-        PythonConfiguration(version='3.7.x', arch="32", identifier='cp37-win32', path='C:\\Python37', nuget_version="3.7.4"),
-        PythonConfiguration(version='3.7.x', arch="64", identifier='cp37-win_amd64', path='C:\\Python37-x64', nuget_version="3.7.4")
+        PythonConfiguration(version='2.7.x', arch="32", identifier='cp27-win32', nuget_version="2.7.17"),
+        PythonConfiguration(version='2.7.x', arch="64", identifier='cp27-win_amd64', nuget_version="2.7.17"),
+        PythonConfiguration(version='3.5.x', arch="32", identifier='cp35-win32', nuget_version="3.5.4"),
+        PythonConfiguration(version='3.5.x', arch="64", identifier='cp35-win_amd64', nuget_version="3.5.4"),
+        PythonConfiguration(version='3.6.x', arch="32", identifier='cp36-win32', nuget_version="3.6.8"),
+        PythonConfiguration(version='3.6.x', arch="64", identifier='cp36-win_amd64', nuget_version="3.6.8"),
+        PythonConfiguration(version='3.7.x', arch="32", identifier='cp37-win32', nuget_version="3.7.5"),
+        PythonConfiguration(version='3.7.x', arch="64", identifier='cp37-win_amd64', nuget_version="3.7.5"),
+        PythonConfiguration(version='3.8.x', arch="32", identifier='cp38-win32', nuget_version="3.8.0"),
+        PythonConfiguration(version='3.8.x', arch="64", identifier='cp38-win_amd64', nuget_version="3.8.0"),
     ]
 
-    if IS_RUNNING_ON_AZURE:
-        # Python 3.4 isn't supported on Azure.
-        # See https://github.com/Microsoft/azure-pipelines-tasks/issues/9674
-        python_configurations = [c for c in python_configurations if c.version != '3.4.x']
-    
     if IS_RUNNING_ON_TRAVIS:
         # cannot install VCForPython27.msi which is needed for compiling C software
         # try with (and similar): msiexec /i VCForPython27.msi ALLUSERS=1 ACCEPT=YES /passive
-        # no easy and stable way fo installing python 3.4 
-        python_configurations = [c for c in python_configurations if c.version != '2.7.x' and c.version != '3.4.x']
+        # no easy and stable way fo installing python 3.4
+        python_configurations = [c for c in python_configurations if c.version != '2.7.x']
```

Comment:
Is this still up to date? At least the comment about 3.4 isn't.
Why can't we get nuget 2.7 on Travis?

Reply:
argh, I forgot I had to drop python 3.4 on this PR.
The issue on travis with 2.7 is not install python but the tools needed to build native code for python 2.7

Reply:
Ah, so it might be good to leave merging this PR until after we drop 3.4?

On the 2.7: is it really necessary to keep worrying about these VS tools for 2.7? I'm building C++17 wheels for Python 2.7 (without the ancient compiler VS offers for 2.7). I've tested this and it does work. The only catch is that "Microsoft Visual C++ Redistributable for Visual Studio 2017" needs to be installed, then, so I [added a section about that to my docs](https://parselmouth.readthedocs.io/en/stable/installation.html#importerror-dll-load-failed-on-windows).

My point being: maybe we could still offer 2.7, but make people aware that the tools won't be installed?

Reply:
IMHO, this goes against the generic wheel principles (I think) which is, pip install should always work, no matter what, no need for further installs. My point is that if we want to offer python 2.7 support on travis, then it shall be an opt-in, not the default. This would require a specific PR.
Anyway, python 2.7 is EOL in less than 2 months. Shall we really do specific things for it ?

Reply:
Hmmm, that's also true, yes. And there's still AppVeyor anyway.

Reply:
> Ah, so it might be good to leave merging this PR until after we drop 3.4?

yes, if we want to merge it after #186 (official vs nuget). My guess is that in a couple months (after python 2.7/3.4) have been dropped, the only difference between the 2 PRs will be what to install and that, with official installers, we'll still need pep514tools vendored-in. The code in `windows.py` will be almost the same, whatever the PR.